src/main/java/com/spotify/docker/client/messages/HostConfig.java

```diff
@@ -1000,6 +1008,10 @@ public BuilderFrom(final String from) {
         this.from = from;
       }
 
+      public BuilderFrom(final Volume volumeFrom) {
```

Comment:
Why `Builder from(final Volume volumeFrom)` on 999 but `BuilderFrom(final Volume volumeFrom)` here instead of `Builder to(final Volume volumeTo)`?